File: src/main/java/org/tindalos/principle/domain/core/logging/TheLogger.java
package org.tindalos.principle.domain.core.logging;

public class TheLogger {
    
    private static ScalaLogger logger;
    
    public static void setLogger(ScalaLogger logger) {
        TheLogger.logger = logger;
    }
    
    public static void info(String msg) {
        logger.info(msg);
    }
    
    public static void error(String msg) {
        logger.error(msg);
    }

}

File: src/main/scala/org/tindalos/principle/domain/core/logging/TheLogger.scala
package org.tindalos.principle.domain.core.logging


object TheLogger {

  var logger: SimpleLogger = _
  
  def setLogger(aLogger:SimpleLogger) = {
    logger = aLogger
  }

  def info(msg: String) = logger.info(msg)
  def error(msg: String) = logger.error(msg)
}